policyengine_us/parameters/calibration/SSI/payments.yaml

```
@@ -0,0 +1,112 @@
+description: Total amount of federally administered SSI payments for 2021 by federal and state
+metadata: 
+  unit: currency-USD (in thousands of dollars)
```

```suggestion
  unit: currency-USD
```

Could we also make the numbers 1-1?